parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -91,6 +91,47 @@ installCriCtlPackage() {
     apt_get_install 20 30 120 ${packageName} || exit 1
 }
 
+installKubeletKubectlPkgFromPMC() {
+    k8sVersion="${1}"
+    installKubePkgWithAptGet "kubelet" "${k8sVersion}" || exit $ERR_KUBELET_INSTALL_TIMEOUT
+    installKubePkgWithAptGet "kubectl" "${k8sVersion}" || exit $ERR_KUBECTL_INSTALL_TIMEOUT
+}
+
+installKubePkgWithAptGet() {
+    packageName="${1:-}"
+    k8sVersion="${2}"
```

Comment:
can we maybe make this function more generatic ? `version`. `installPkgWithAptGet`